src/app/rosetta/docker-start.sh

```diff
@@ -47,9 +47,12 @@ DEFAULT_FLAGS="--config-dir ${MINA_CONFIG_DIR} --peer-list-url ${PEER_LIST_URL}
 export MINA_FLAGS=${MINA_FLAGS:=$DEFAULT_FLAGS}
 # Postgres database connection string and related variables
 POSTGRES_USERNAME=${POSTGRES_USERNAME:=pguser}
-POSTGRES_DBNAME=${POSTGRES_DBNAME:=archive}
+POSTGRES_DBNAME=${POSTGRES_DBNAME:=archive_balances_migrated}
+POSTGRES_PORT=${POSTGRES_PORT:=5432}
+POSTGRES_HOST=${POSTGRES_HOST:=127.0.0.1}
+POSTGRES_PASSWORD=${POSTGRES_PASSWORD:=pguser}
 POSTGRES_DATA_DIR=${POSTGRES_DATA_DIR:=/data/postgresql}
-PG_CONN=postgres://${POSTGRES_USERNAME}:${POSTGRES_USERNAME}@127.0.0.1:5432/${POSTGRES_DBNAME}
+PG_CONN=postgres://${POSTGRES_USERNAME}:${POSTGRES_PASSWORD}@${POSTGRES_DATA_DIR}:${POSTGRES_PORT}/${POSTGRES_DBNAME}
```

Comment:
this seems sus but OK

Reply:
I think there is an issue in the hostname: `127.0.0.1` should not be replaced by `${POSTGRES_DATA_DIR}`

Reply:
The code from compatible was commited with this mistake, instead of `${POSTGRES_DATA_DIR}` we should have `${POSTGRES_HOST}`